.github/workflows/End2EndTestApache.yml

```diff
@@ -14,13 +14,15 @@ jobs:
       matrix:
         apache_kafka_version: [ '2.5.1', '2.8.1', '3.2.1' ]
         snowflake_cloud: [ 'AWS', 'AZURE', 'GCS' ]
+        java_test_version: [ '8', '11', '17' ]
     steps:
     - name: Checkout Code
       uses: actions/checkout@v2
     - name: "Install Java 8"
```

Comment:
why do we need this step? as in we are installing java 8 here but using different versions below.

Reply:
the whole point of my change is to use two different versions of java - java 8 for building and unit testing but then 8, 11 and 17 as jre to run the app and verify it against given runtime environment